config/localTemplate.go

```diff
@@ -149,29 +153,35 @@ type Local struct {
 	// outgoing broadcast messages from this node.
 	PriorityPeers map[string]bool `version[4]:""`
 
-	// To make sure the algod process does not run out of FDs, algod ensures
+	// ReservedFDs is used to make sure the algod process does not run out of file descriptors (FDs). Algod ensures
 	// that RLIMIT_NOFILE >= IncomingConnectionsLimit + RestConnectionsHardLimit +
 	// ReservedFDs. ReservedFDs are meant to leave room for short-lived FDs like
 	// DNS queries, SQLite files, etc. This parameter shouldn't be changed.
 	// If RLIMIT_NOFILE < IncomingConnectionsLimit + RestConnectionsHardLimit + ReservedFDs
 	// then either RestConnectionsHardLimit or IncomingConnectionsLimit decreased.
 	ReservedFDs uint64 `version[2]:"256"`
 
-	// local server
-	// API endpoint address
+	// EndpointAddress configures the address the node listens to for REST API calls. Specify an IP and port or just port. For example, 127.0.0.1:0 will listen on a random port on the localhost (preferring 8080).
 	EndpointAddress string `version[0]:"127.0.0.1:0"`
 
-	// timeouts passed to the rest http.Server implementation
-	RestReadTimeoutSeconds  int `version[4]:"15"`
+	// RestReadTimeoutSeconds is passed to the API servers rest http.Server implementation.
+	RestReadTimeoutSeconds int `version[4]:"15"`
+
+	// RestWriteTimeoutSeconds is passed to the API servers rest http.Server implementation.
 	RestWriteTimeoutSeconds int `version[4]:"120"`
 
-	// SRV-based phonebook
+	// DNSBootstrapID specifies the names of a set of DNS SRV records that identify the set of nodes available to connect to.
+	// When resolving the bootstrap ID <network> will be replaced by the genesis block's network name. This string uses a URL
+	// parsing library and supports optional backup and dedup parameters. 'backup' is used to provide a second DNS entry to use
+	// in case the primary is unavailable. dedup is intended to be used to deduplicate SRV records returned from the primary
+	// and backup DNS address. If the <name> macro is used in the dedup mask, it must be at the beginning of the expression.
+	// This is not typically something a user would configure. For more information see config/dnsbootstrap.go.
```

Comment:
@algogm DNSBootstrapID was one of the mis-documented settings. I copied over some of the highlights from dnsbootstrap.go, how does this look?